lib/trento/domain/host/events/host_registered.ex

```diff
@@ -9,7 +9,6 @@ defmodule Trento.Domain.Events.HostRegistered do
     field :host_id, Ecto.UUID
     field :hostname, :string
     field :ip_addresses, {:array, :string}
```

Comment:
We need to agree on when we will upcast or, in general, what we consider a breaking change. This is valid for API versioning but also event versioning (internal events and integration events).
My proposal is to go like this but track it somewhere.

Reply:
@fabriziosestito I will create a ticket in our backlog to discuss this.
I think that we might agree that in the particular scenario (removing a field), we don't have any breaking change, so upgrading the version would only be "informative"